configure/RULES_BUILD

```diff
@@ -532,11 +532,11 @@ endif # LOADABLE_SHRLIB_SUFFIX
 ifneq ($(INSTALL_CONFIGS),)
 $(INSTALL_CONFIG)/%: %
 	$(ECHO) "Installing config file $@"
-	@$(INSTALL) -d -m $(INSTALL_PERMISSIONS) $< $(@D)
+	@$(INSTALL) -d -m $(INSTALL_PERMISSIONS) $(abspath $<) $(abspath $(@D))
```

Comment:
The `$(abspath ...)` function accepts multiple inputs, so the command could be shortened to just this:
```makefile
	@$(INSTALL) -d -m $(INSTALL_PERMISSIONS) $(abspath $< $(@D))
```

This doesn't support source or installation paths containing spaces, but I doubt that this is the only place where we'd need to add double-quotes around filenames if we wanted that.

Reply:
Good catch. I was thinking of abspath implicitly working that way for all of the targets `$<` (although I believe in this case that will never be more than one), but I didn't think of putting the directory there too. Fixed.